rules_building_block/persistence_cap_sys_admin_added_to_new_binary.toml

```diff
@@ -0,0 +1,82 @@
+[metadata]
+bypass_bbr_timing = true
+creation_date = "2024/01/10"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "Linux effective and permitted process capability data sources were added in version 8.11.0"
+min_stack_version = "8.11.0"
+updated_date = "2024/01/10"
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+Identifies instances where a binary is granted the CAP_SYS_ADMIN capability. In Linux, the CAP_SYS_ADMIN capability is a
```

Comment:
May want to add something that explains the logic behind your use of new_terms for this rule